src/Support/SupportData.php

```diff
@@ -132,7 +127,7 @@ public function parse_args() {
 			}
 		}
 
-		$this->urls = array_map( __CLASS__ . '::normalize_url_for_storage', $this->urls );
+		$this->urls = array_map( AMP_Validated_URL_Post_Type::class . '::normalize_url_for_storage', $this->urls );
```

Comment:
As @westonruter suggested in https://github.com/ampproject/amp-wp/pull/6147#discussion_r707765434, it would be better to use a closure here (and in other places throughout this class), especially for static analysis purposes.


```suggestion
		$this->urls = array_map(
			static function ( $url ) {
				return AMP_Validated_URL_Post_Type::normalize_url_for_storage( $url );
			},
			$this->urls 
		);
```